docs/Datadog.Trace.Bundle/README.md

```diff
@@ -0,0 +1,93 @@
+- You can use it in locations where you cannot access the underlying host to install the MSI or Linux package.
+- You can have multiple applications on the same host using different versions of Datadog.Trace.Bundle without issue.
+
+## Getting Started
+
+1. Configure the Datadog agent for APM [as described in our documentation](https://docs.datadoghq.com/tracing/setup_overview/setup/dotnet-core#configure-the-datadog-agent-for-apm).
+2. Add the [Datadog.Trace.Bundle](https://www.nuget.org/packages/Datadog.Trace.Bundle) NuGet package to your project, using `dotnet add package Datadog.Trace.Bundle`, for example.
+3. Configure the tracer on the server, as [described below](#configure-the-tracer)
+4. [View your live data on Datadog](https://app.datadoghq.com/apm/traces).
+
+### Configure the tracer
+
+After adding the NuGet package to your project, set the following **required** environment variables to enable automatic instrumentation of your application.
+
+> **_NOTE:_** 
+The following are the mandatory variables. More options are available in our public documentation for the [Tracer](https://docs.datadoghq.com/tracing/trace_collection/library_config/dotnet-core/?tab=environmentvariables) and the [Continuous Profiler](https://docs.datadoghq.com/profiler/enabling/dotnet/?tab=linux#configuration) to tune your usage. 
+
+
+.NET Core:
+
+```
+CORECLR_ENABLE_PROFILING=1
+CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
+CORECLR_PROFILER_PATH=<System-dependent path>
+DD_DOTNET_TRACER_HOME=<APP_DIRECTORY>/datadog
+```
+
+.NET Framework:
+
+```
+COR_ENABLE_PROFILING=1
+COR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
+COR_PROFILER_PATH=<System-dependent path>
+DD_DOTNET_TRACER_HOME=<APP_DIRECTORY>/datadog
+```
+
+The value for the `<APP_DIRECTORY>`  placeholder is the path to the directory containing the application’s .dll files. The value for the `CORECLR_PROFILER_PATH` environment variable varies based on the system where the application is running:
+
+| OPERATING SYSTEM AND PROCESS ARCHITECTURE      | CORECLR_PROFILER_PATH VALUE |
+| ----------- | ----------- |
+| Alpine Linux x64      | <APP_DIRECTORY>/datadog/linux-musl-x64/Datadog.Trace.ClrProfiler.Native.so       |
+| Linux x64   | <APP_DIRECTORY>/datadog/linux-x64/Datadog.Trace.ClrProfiler.Native.so        |
+| Linux ARM64      | <APP_DIRECTORY>/datadog/linux-arm64/Datadog.Trace.ClrProfiler.Native.so       |
+| Windows x64   | <APP_DIRECTORY>\datadog\win-x64\Datadog.Trace.ClrProfiler.Native.dll        |
+| Windows x86      | <APP_DIRECTORY>\datadog\win-x86\Datadog.Trace.ClrProfiler.Native.dll       |
+
+For Docker images running on Linux, configure the image to run the createLogPath.sh script:
+
+```
+RUN /<APP_DIRECTORY>/datadog/createLogPath.sh
+```
+
+Docker examples are available [here](https://github.com/DataDog/dd-trace-dotnet/tree/master/tracer/samples/NugetDeployment)
+
+For standalone applications, manually restart the application.
```

Comment:
Two thoughts here:
- The docker examples, could this go in a separate section or header?
- I think we can remove the restart the application line if we take my suggestion to move the wording to the beginning of the section

Reply:
> I think we can remove the restart the application line if we take my suggestion to move the wording to the beginning of the section

I hesitated to remove it as well actually (for the same reasons as above). So I'll remove it here anyway, and depending on the discussion, I'll add it above

Reply:
Done in 05964fbfe0871542759b1d5d2cf1b742f6af58f6